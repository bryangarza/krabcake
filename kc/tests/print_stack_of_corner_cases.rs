use test_dependencies::{kc_borrow_mut, print_stack_of};

fn main() {
    let mut percent_s = 5;
    let p = kc_borrow_mut!(percent_s);
    let msg = b"C format sequences are %% %b %q %d %i %o %u %x %X %f %e %E %g %G %c %s\0";
    print_stack_of(msg.as_ptr(), p);
}
