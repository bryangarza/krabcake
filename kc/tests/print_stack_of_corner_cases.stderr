lib.rs: normalizing output...
Hello world (from `rs_hello/src/lib.rs`)!
kc_main.c: dispatching code 4b430000
lib.rs: handle client request BORROW_MUT. borrowing_addr: 0x10000000
kc_main.c: dispatching code 4b43000a
print_stack_of `C format sequences are %% %b %q %d %i %o %u %x %X %f %e %E %g %G %c %s` (0x10000000): [Unique(Counter(Counter(1)))]
